tnoodle-ui/src/wca/events.js

```diff
@@ -1,14 +1,361 @@
+        name: "Clock",
+        format_ids: ["a", "3", "2", "1"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      minx: {
+        id: "minx",
+        name: "Megaminx",
+        format_ids: ["a", "3", "2", "1"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      pyram: {
+        id: "pyram",
+        name: "Pyraminx",
+        format_ids: ["a", "3", "2", "1"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      skewb: {
+        id: "skewb",
+        name: "Skewb",
+        format_ids: ["a", "3", "2", "1"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      sq1: {
+        id: "sq1",
+        name: "Square-1",
+        format_ids: ["a", "3", "2", "1"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      "444bf": {
+        id: "444bf",
+        name: "4x4x4 Blindfolded",
+        format_ids: ["3", "2", "1"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      "555bf": {
+        id: "555bf",
+        name: "5x5x5 Blindfolded",
+        format_ids: ["3", "2", "1"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      "333mbf": {
+        id: "333mbf",
+        name: "3x3x3 Multi-Blind",
+        format_ids: ["3", "2", "1"],
+        can_change_time_limit: false,
+        is_timed_event: false,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: true
+      },
+      magic: {
+        id: "magic",
+        name: "Magic",
+        format_ids: ["a"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      mmagic: {
+        id: "mmagic",
+        name: "Master Magic",
+        format_ids: ["a"],
+        can_change_time_limit: true,
+        is_timed_event: true,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: false
+      },
+      "333mbo": {
+        id: "333mbo",
+        name: "3x3x3 Multi-Blind Old Style",
+        format_ids: ["3", "2", "1"],
+        can_change_time_limit: false,
+        is_timed_event: false,
+        is_fewest_moves: false,
+        is_multiple_blindfolded: true
+      }
+    },
+    extend
+  )
 };
 
 function extend(rawEvent) {
   rawEvent = _.mapKeys(rawEvent, (v, k) => _.camelCase(k));
   return {
-    ...rawEvent,
-  }
+    ...rawEvent
```

Comment:
Was it removed automatically by your text editor?
In the website source we try to enforce the trailing commas in multilines arrays and map, it leads to cleaner diff when adding/removing elements.